input/metadata.json

```
@@ -0,0 +1,1349 @@
+        "properties": {
+          "type": "number",
+          "minimum": 1,
+          "maximum": 300
+        }
+      },
+      "lac_cow_feeds": {
+        "type": "array",
+        "description": "Lactating Cow Feeds -- Add as many lactating cow feeds as needed",
+        "properties": {
+          "type": "number",
+          "minimum": 1,
+          "maximum": 300
+        }
+      },
+      "purchased_feeds": {
+        "type": "array",
+        "description": "Purchased Feeds -- Select one purchased feed at a time and add the cost per kg of dry matter",
+        "properties": {
+          "type": "object",
+          "purchased_feed": {
+            "type": "number",
+            "description": "Purchased Feed",
+            "minimum": 1,
+            "maximum": 300
+          },
+          "purchased_feed_cost": {
+            "type": "number",
+            "description": "Cost of Feed (US dollars/kg DM)",
+            "minimum": 0
+          }
+        }
+      },
+      "farm_grown_feeds": {
+        "type": "array",
+        "description": "Farm-Grown Feeds",
+        "properties": {
+          "type": "number",
+          "minimum": 1,
+          "maximum": 300
+        }
+      },
+      "storage_options": {
+        "type": "array",
+        "description": "Storage Options -- Add as many storage options as needed",
+        "properties": {
+          "type": "object",
+          "storage_type": {
+            "type": "string",
+            "description": "Storage Type -- The type of storage unit used. This only applies to farm grown forage feeds",
+            "default": "Bunker Silo",
+            "pattern": "^{8' Diameter Bag|10' Diameter Bag|12' Diameter Bag|14' Diameter Bag|Upright Silo - Traditional|Upright Silo - Oxygen-Excluding|Bunker Silo|Pile|Wrapped Bale|Bale - Covered|Bale - Uncovered|Bin}$"
+          },
+          "moisture": {
+            "type": "string",
+            "description": "Moisture -- The management practices used that determine the dry matter content of the preserved feed.",
+            "default": "Wilted",
+            "pattern": "^{Direct Cut|Wilted|Baleage|Haylage|Moist Hay|Dry Hay}$"
+          },
+          "additive": {
+            "type": "string",
+            "description": "Additive -- Type of additive added to the forage material before ensiling.",
+            "pattern": "^{preservative|nutrient additive|absorbant}$",
+            "default": "preservative"
+          },
+          "package_density": {
+            "type": "number",
+            "description": "Package Density -- The density of ensiled feed material, kg DM /m3",
+            "minimum": 200,
+            "maximum": 800,
+            "default": 14
+          },
+          "inoculation": {
+            "type": "string",
+            "description": "Inoculation",
+            "pattern": "^{None|heterofermentative|homofermentative}$"
```

It looks like none of the radio fields are generating the auto-default. Because there are "defaults" in the DCA feeds section, I think it could be helpful to have the same default here as well.
```suggestion
            "type": "string",
            "description": "Inoculation",
            "pattern": "^{None|heterofermentative|homofermentative}$",
            "default": null
```

---

I think maybe we should default to the None option?
But good point! I will make sure all enums have a default.